app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
```

Comment:
maybe it's better just from app import errors

Reply:
There should be an empty line between the imports of default libraries and custom files.